tools/resize-observer-test-error-handler.js

```diff
@@ -1,15 +1,5 @@
-// eslint-disable-next-line no-undef
-before(() => {
-	const e = window.onerror;
-	window.onerror = function(err) {
-		if (err === 'ResizeObserver loop limit exceeded') {
-			console.warn('Ignored: ResizeObserver loop limit exceeded');
-			return false;
-		} else if (err === 'ResizeObserver loop completed with undelivered notifications.') {
-			console.warn('Ignored: ResizeObserver loop completed with undelivered notifications');
-			return false;
-		} else {
-			return e(...arguments);
-		}
-	};
```

Comment:
I'd originally deleted this and moved it inline into the configuration, but then realized that the [create initializer](https://github.com/BrightspaceUI/create/blob/main/src/generators/test-unit/templates/static/karma.conf.js#L14) uses it which means lots of folks' projects are using it.

The original way this was written by hooking into the `before` hook no longer worked, but I found this to work equally well.